### DISCUSSION TITLE
Aderyn Report problem

### DISCUSSION THREAD

**Discussion by msyuvaraj** [Cyfrin/security-and-auditing-full-course-s23#149] [Q&A]
hey,
when i run `aderyn .` in the terminal there is an error regarding some ast not found but the report is generated which have zero issues.
I have attached the screenshot please find it and help in this regard.
![aderynSC](https://github.com/Cyfrin/security-and-auditing-full-course-s23/assets/111735823/11301dc3-4c17-4bb6-90c0-437ebbbbc012)

**Comment by EngrPips** [comment]
Hello @msyuvaraj , glad you report this. I run into this same issue myself and i haven't be able to resolve it. I would suggest maybe you re-clone the codebase , rebuild it and retry running aderyn on it, I would be looking forward to hear back from you.

**Comment by alymurtazamemon** [comment]
@msyuvaraj I think it is looking for some compiled JSON files. Try to clean your project, re-compile it and then try it.

```bash
forge clean
forge build
aderyn .
```

**Comment by PatrickAlphaC** [comment]
Could you put this issue into the aderyn codebase? https://github.com/Cyfrin/aderyn
